MAUI/AIAssistView/getting-started.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title: Getting Started with .NET MAUI AI Assist View Control | Syncfusion
+description: Learn here about getting started with the Syncfusion .NET MAUI AI Assist View (SfAIAssistView) control, its elements and more.
+platform: MAUI
+control: SfAIAssistView
+documentation: ug
+---
+
+# Getting Started with .NET MAUI AI Assist View (SfAIAssistView)
+
+This section provides a quick overview for getting started with the .NET MAUI AI Assist View (SfAIAssistView). Walk-through the entire process of creating real world of this control.
+
+## Creating an application using the .NET MAUI SfAIAssistView
```

Comment:
here say control name as "AI AssistView",
follow the same for all headings.

Reply:
ok sir